roles/kubernetes-apps/external_provisioner/local_path_provisioner/defaults/main.yml

```diff
@@ -6,3 +6,4 @@ local_path_provisioner_reclaim_policy: Delete
 local_path_provisioner_claim_root: /opt/local-path-provisioner/
 local_path_provisioner_is_default_storageclass: "true"
 local_path_provisioner_debug: false
+local_path_provisioner_helper_image_tag: "latest"
```

Comment:
Shouldn't `local_path_provisioner_helper_image_repo` be defined here as well?

Reply:
image is defined by default, if we not define local_path_provisioner_helper_image_repo local path provisioner will just use default busybox container, but if we define only local_path_provisioner_helper_image_repo and not local_path_provisioner_helper_image_tag - it may cause to inaccessible of container in some registrys.